src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -2068,6 +2090,22 @@ public void subscribe(String topicName, String topicMethod) {
     String callbackMethod = CodecUtils.getCallbackTopicName(topicMethod);
     subscribe(topicName, topicMethod, getName(), callbackMethod);
   }
+  
+  public void subscribeTo(String service, String method) {
+    subscribe(service, method, getName(), CodecUtils.getCallbackTopicName(method));
+  }
+  
+  public void subscribeToRuntime(String method) {
+    subscribe(Runtime.getInstance().getName(), method, getName(), CodecUtils.getCallbackTopicName(method));
+  }
+  
+  public void unsubscribeTo(String service, String method) {
+    unsubscribe(service, method, getName(), CodecUtils.getCallbackTopicName(method));
+  }
+  
+  public void unsubscribeToRuntime(String method) {

```

Comment:
just a nit pick.. unsubscribeFromRuntime might be a slightly better method name.